src/qps.cpp

```diff
@@ -1009,7 +1009,7 @@ void Qps::make_command_menu()
     }
     connect(m_command, SIGNAL(triggered(QAction *)),
             SLOT(run_command(QAction *)));
-
+*/
```

Comment:
Please don't comment out these for now.

Qps is full of strange code lines. The "Command" menu is one of them. Although it only has the Quit action, we might be able to read the mind of the original author at some point. Commenting out lines like these will make the job harder later.

Reply:
Ok I have just commented out line 992, whith a message to explain why.